routes/api/v1/forecast.js

```diff
@@ -0,0 +1,68 @@
+var express = require('express');
+var router = express.Router();
+const fetch = require('node-fetch');
+const dotenv = require('dotenv');
+dotenv.config();
+const { darkSkyApi, googleApi } = require('../../../config/config');
+pry = require('pryjs');
+
+class Forecast {
```

Comment:
I like this OOP approach to building the Forecast. You could move this class to another file and then just `require` it up top.  It might reduce the amount of logic in the route and then could also be reused elsewhere.

Reply:
I was just looking into that I will see if I can get that knocked out.